compose/xui-frontend.yml

```diff
@@ -0,0 +1,63 @@
+---
+version: '2.4'
+
+services:
+  xui-manage-cases:
+    image: "${XUI_MANAGE_CASES_USE_LOCAL-hmctspublic.azurecr.io/}xui/webapp:${XUI_MANAGE_CASES_TAG:-latest}"
+    container_name: xui-manage-cases
+    environment:
+      FEATURE_APP_INSIGHTS_ENABLED: "true"
+      FEATURE_SECURE_COOKIE_ENABLED: "false"
+      FEATURE_PROXY_ENABLED: "false"
+      FEATURE_TERMS_AND_CONDITIONS_ENABLED: "false"
+      FEATURE_HELMET_ENABLED: "true"
+      FEATURE_OIDC_ENABLED: "false"
+      FEATURE_REDIS_ENABLED: "false"
+
+      MICROSERVICE: xui_webapp
+      PROTOCOL: http
+      XUI_ENV: local
+      NODE_CONFIG_ENV: development
+
+      SERVICES_CCD_COMPONENT_API: http://localhost:3453
+      SERVICES_CCD_DATA_STORE_API: http://ccd-data-store-api:4452
+      SERVICES_IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      SERVICES_IDAM_CLIENT_ID: xuiwebapp
+      SERVICES_IDAM_LOGIN_URL: "${IDAM_STUB_LOCALHOST:-http://localhost:3501}"
+      SERVICES_IDAM_INDEX_URL: /
+      SERVICES_IDAM_OAUTH_CALLBACK_URL: /oauth2/callback
+      SERVICES_S2S: http://service-auth-provider-api:8080
+      REDISCLOUD_URL: http://localhost:6780
+
+      HEALTH_CCD_COMPONENT_API: http://localhost:3453/health
+      HEALTH_CCD_DATA_API: http://ccd-data-store-api:4452/health
+
+      APPINSIGHTS_INSTRUMENTATIONKEY: TESTVAR
+      IDAM_SECRET: xuiwebapp_secrect
+      S2S_SECRET: "${BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP}"
```

Comment:
Not defaulting to anything? How do we know what should be the value in local?

Reply:
this is similar to **DEFINITION_STORE_IDAM_KEY: "${IDAM_KEY_CCD_DEFINITION_STORE}"**
We have defined BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP in **.env** file

Reply:
Or is it just for the secret like in here for example: https://github.com/hmcts/aac-manage-case-assignment/blob/master/Jenkinsfile_CNP#L32 ?